rootfs/etc/systemd/system/cec-poweroff-tv.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Use CEC to power off TV
+DefaultDependencies=no
+Before=final.target
+
+[Service]
+Type=oneshot
+ExecStart=bash -c "echo 'standby 0' | cec-client -s"
+
+[Install]
+WantedBy=poweroff.target
```

Comment:
Standby 0 does not power off the TV, it marks the input as in standby. The TV (and mine does do this, tested using Kodi which has libCEC) will just switch to another active input and only suspends if evetr input is on standby.

I.e., it you have your Xbox on screen and suddenly your chimeraOS box issues a standby, the TV won't turn off, it will just internally mark that the chimeraOS input is in standby.